controllers/topics.controllers.js

```diff
@@ -1,8 +1,14 @@
-const { fetchTopics } = require("../models/topics.models");
+const { fetchTopics, fetchArticle } = require("../models/topics.models");
```

Comment:
Please make a separate model and controller for articles! I'd say that, as a rule of thumb, if it's a table in the db, it should have it's own model and controller 👍